src/io/fasta.rs

```diff
@@ -1460,12 +1460,20 @@ ATTGTTGTTTTA
         let expected = io::Error::new(io::ErrorKind::NotFound, "foo");
 
         assert_eq!(actual.kind(), expected.kind());
-        assert!(actual.to_string().starts_with("No such file or directory"))
+
+        #[cfg(unix)]
+        assert!(actual.to_string().starts_with("No such file or directory"));
+
+        #[cfg(windows)]
+        assert!(actual
+            .to_string()
+            .starts_with("The system cannot find the path specified"));
```

Comment:
I used this to fix the windows errors, but... ugh. I think it's better to just remove the `.starts_with()` checks, because this is going to break if other systems show up...

Reply:
I removed these checks in https://github.com/rust-bio/rust-bio/pull/277/commits/48a548906a9f8ad712d73676fc87fc414c7eb224, since the important thing is matching the error kind.